bin/BuildPackages.sh

```diff
@@ -85,6 +98,7 @@ make TOPDIR=`pwd`
 chmod -R a+rX .
 cd bin
 aa=`./config.guess`
+# TODO dangerous to parse output of ls, use find instead
```

Comment:
So let's resolve this TODO... First off, `aa` is a horrible name. Better would be something like `hostarch`... But even that is problematic, because the `config.guess` we are invoking here might differ from the one GAP uses. The correct solution would be 
```sh
source "$GAPDIR/sysinfo.gap
```
and then use the variable `GAParch` set by it. Then we can simply refer to `$GAPDIR/bin/$GAParch` and need neither `ls` nor `find`.